.github/workflows/e2e-chrome.yml

```diff
@@ -24,6 +24,7 @@ jobs:
 
   test-e2e-chrome-webpack:
     uses: ./.github/workflows/run-e2e.yml
+    if: ${{ github.event_name != 'merge_group' }}
```

Comment:
Interesting. At first I was thinking, wouldn't this be `workflow_call`? But the docs seem to indicate that the `github` context is set to the _caller_'s context when a reusable workflow is called. TIL.